docs/connected_orgs.rst

```diff
@@ -1,14 +1,133 @@
-Connect persistent orgs
+Connect Persistent Orgs
 =======================
 
-Connect a production org
-------------------------
+In addition to creating `scratch orgs<TODO>`_, you can connect persistent orgs to your CumulusCI project to run tasks and flows on them. This feature supports use cases such as deploying to a Developer Edition org to release a package version, or installing to a sandbox for user acceptance testing.
```

Comment:
So we don't actually say much about what a persistent org is and only touch up a little bit about why they're helpful to use in ``cci``. Does it just mean an org that isn't a DX scratch org?

Reply:
Yes. This sentence is attempting to clarify that by giving examples of types of persistent orgs (Developer Edition and sandboxes) that the reader is more likely to be familiar with than scratch orgs. A customer's production org is another type of persistent org.